buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/ReportGenerationPerformanceTest.groovy

```diff
@@ -31,6 +31,9 @@ import org.openmbee.junit.model.JUnitTestSuite
 
 @CompileStatic
 abstract class ReportGenerationPerformanceTest extends PerformanceTest {
```

Comment:
Is this the only subclass of `PerformanceTest` now? Maybe we should merge the two classes again.

Reply:
This is an abstract class providing contract for its subclasses. I'd like keep it.